lemmatization/services/latin.py

```diff
@@ -159,8 +143,8 @@ def preprocessor(self, tokens):
                     for enclitic in LATIN_ENCLITICS:
                         if word.endswith(enclitic) and found_first_enclitic is False:
                             found_first_enclitic = True
-                            #Creates example word, word_normalized, following = new_token
-                            new_tokens.append((word[:word.find(enclitic)], enclitic, ""))
+                            word_without_enclitic = word[:word.find(enclitic)]
```

Comment:
Agreed with your points @elliottyates There could be the edge case that you pointed out where the `enclitic pattern` would appear other places(names?) but I am not totally sure. @bilbe might know more about this. I think for now we are expecting the enclitics to only appear at the end of the word which this piece of code(`word[:-len(enclitic)]`) should do to slice off the enclitic.

Reply:
The `word[:-len(enclitic)]` approach makes sense to me, too. Maybe to handle proper nouns `word_without_enclitic` should go through `lookup_form()` as well, and if there's no found form with the enclitic removed, it just gives up and returns the token as is? I feel like any time I'm not sure if there's an edge case in a language, there is, especially with user-generated input.

Reply:
Great catch @elliottyates @Tanvez @jaguillette . Thank you for spotting this! Yes, this is definitely an edge case that we need to handle. Instead of searching from the start of the string for the first occurrence of the enclitic, we could start from the end and work backward as suggested since the enclitic is a known string. I agree as well that we might erroneously split a token that appears to have the enclitic, even though it's not, such as in a person's name. Good idea @jaguillette to only split if we can positively identify the form, otherwise leave it unchanged. I'm going to rework this with all of your suggestions.